src/App.js

```diff
@@ -3,15 +3,18 @@ import { useEffect } from "react"
 import { Routes, Route } from "react-router-dom";
 import Mockman from "mockman-js";
 import { Header, Sidebar, SignIn, SignUp } from "components/index"
-import { NotFound, LandingPage, HistoryPage, WatchLaterPage, LikedPage, PlayListPage, VideoPage, AuthorizationPage, RestrictAuth } from "pages/index"
+import { NotFound, LandingPage, HistoryPage, WatchLaterPage, LikedPage, PlayListPage, VideoPage, AuthorizationPage, RestrictAuth, IndividualPlayListPage } from "pages/index"
 import { ToastContainer } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import { useAuth } from 'context/authContext';
+import { usePlayList } from 'context/playListContext';
+import { PlayListModal } from 'components/playListModal/PlayListModal';
```

Comment:
You are also importing Header,SignIn, and Signup from the components folder so it would be much better if you add PlaylistModal to index.js of components and import all of them in one line.

Reply:
Yeah, this was auto imported by VS Code. Will change it. Thanks for pointing out!